### ISSUE TITLE
breaking bamboo does not proc infinitetools

### ISSUE THREAD

**Issue by westlyroots** [Earthcomputer/clientcommands#111]
after cracking the player RNG and enabling the temp rule, breaking bamboo still removes damage from swords.

**Comment by leumasme** [comment]
As Bamboo breaks instantly with a sword, this may be #59 (or does it not drop items at all?)

**Comment by westlyroots** [comment]
> As Bamboo breaks instantly with a sword, this may be #59 (or does it not drop items at all?)

This does not seem to be related to specifically instant breaking. InfiniteTools still works with an eff V pick in the nether, even if I mine for minutes at a time. blocks still break properly, and I never notice infinitetools drop items.

**Comment by Earthcomputer** [comment]
Yeah I think #59 is fixed in vanilla by now, let me close that one

**Comment by westlyroots** [comment]
note: this seems to not happen with axes and chorus plants.
